lib/dash/segment_template.js

```diff
@@ -85,7 +85,9 @@ shaka.dash.SegmentTemplate = class {
         },
       };
     } else if (info.segmentDuration) {
-      if (!isUpdate && context.adaptationSet.contentType !== 'image') {
+      if (!isUpdate &&
+        context.adaptationSet.contentType !== 'image' &&
```

Comment:
style nit: indent +2 more here (+4 total for line continuation)

I thought our linter would be as picky as me about this... 😁

Reply:
@joeyparrish Can you create a rule so this doesn't happen again?

Reply:
I'm sure it's in eslint somewhere.  I don't have time to dig into it right now, and the impact is very minor.